space2stats_api/src/space2stats/api/handler.py

```diff
@@ -5,14 +5,18 @@
 
 from mangum import Mangum
 
-from .app import app
+from ..settings import Settings
+from .app import build_app
 from .db import connect_to_db
 
+settings = Settings(DB_MAX_CONN_SIZE=1)  # disable connection pooling
```

Comment:
Remind me to chat about: 
> Made it so that the Mangum handler would always specify a max connection of 1 to reduce load on the DB (lambdas can only handle one concurrent request)

Just curious to understand it in more detail.

Reply:
Thanks for the explanation. Each lambda can only have a single connection at once.